CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
+## 0.53.0 / 2021-12-15
```

Comment:
The https://github.com/prometheus-operator/prometheus-operator/pull/4323 is missing.

Do we also want to mention that default repo branch was changed in https://github.com/prometheus-operator/prometheus-operator/pull/4384 ?

Reply:
done for #4323 thanks! I don't feel like the changelog should mention the default branch change since it has no effect on users?